test/functional/checkout.js

```diff
@@ -109,4 +131,56 @@ describe('Check out items', () => {
         });
     });
 
+    it('should check out a model', () => {
+        assert.lengthOf(CheckoutStore.getCheckouts(), 1);
+        assert.strictEqual(ModelStore.getModelByAddress('m8y7nFLsT').count, 100);
```

Comment:
@thofmann Because in a majority of our other tests we don't hard-code the addresses. We call items[0].address instead which can be used here. Plus if we were to ever change how model addresses are generated this test would have a chance of failing.